embark-org.el

```diff
@@ -273,6 +278,7 @@ what part or in what format the link is copied."
 (embark-org-define-link-copier description description "'s description")
 (embark-org-define-link-copier target target "'s target")
 
+(declare-function embark-org-copy-link-inner-target "embark-org")
```

Comment:
Oh, this one surprises me, I didn't think I'd need to declare this function since it's definition is one line below!

Reply:
Me too :stuck_out_tongue_closed_eyes: